src/Tizen.Multimedia.Remoting/WebRTC/MediaFileSource.cs

```diff
@@ -15,23 +15,21 @@
  */
 
 using System;
-using System.ComponentModel;
 using System.Diagnostics;
 using NativeWebRTC = Interop.NativeWebRTC;
 
 namespace Tizen.Multimedia.Remoting
 {
     /// <summary>
-    /// Represents a file source.
+    /// Represents a media source with contents read from a file.
     /// </summary>
     /// <remarks>
-    /// The media storage privilege(http://tizen.org/privilege/mediastorage) is required.<br/>
-    /// The external storage privilege(http://tizen.org/privilege/externalstorage) is required.
+    /// Depending on where the source file is located either the media storage privilege (http://tizen.org/privilege/mediastorage) is required or<br/>
+    /// the external storage privilege(http://tizen.org/privilege/externalstorage) is required
```

Comment:
the dot at the end of the sentence disappeared. Please restore it

Reply:
I'll raise new PR because of git problem which can not be recorverred.